flytekit/image_spec/image_spec.py

```diff
@@ -153,9 +158,9 @@ def exist(self) -> bool:
                     f"    pip install --upgrade docker"
                 )
 
-            click.secho(f"Failed to check if the image exists with error : {e}", fg="red")
-            click.secho("Flytekit assumes that the image already exists.", fg="blue")
```

Comment:
Can we ensure we handle runtime error nicely in the clean exception in pyflyte

Reply:
updated